.travis.yml

```diff
@@ -33,6 +33,8 @@ install:
       pip install -r requirements2.txt;
     else
       pip install -r requirements3.txt;
+      wget http://launchpadlibrarian.net/151932048/libc6_2.17-0ubuntu5.1_i386.deb;
```

Comment:
Yeah, that error is going to be messy to sort out.  But I had immediate concerns with your attempted fix for two reasons.  First `libc` is super fundamental basic dependency that is typically already present ... if you're encountering a problem with versions of libc you're usually in some sort of hot water and must proceed with caution.  Second, the package you installed had `386` in the name ... which indicates it's for 32bit linux ... you're almost certainly running on a 64bit platform.

Good luck!!!!